core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20250228_add_billing_project_id.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.30.xsd">
+    <changeSet id="add_billing_project_id" author="kmarete" logicalFilePath="dummy">
+        <addColumn tableName="BILLING_PROJECT">
+            <column name="ID" type="BINARY(16)">
+                <constraints nullable="false" />
+            </column>
+        </addColumn>
+
+        <!-- Populate ID column with UUID derived from NAME (Unique PK) -->
+        <update tableName="BILLING_PROJECT">
+            <column name="ID" valueComputed="UNHEX(MD5(NAME))"/>
```

Comment:
This seems like a fine way to seed the ids, but there is also a `UUID()` function in mysql that you can use to generate a random UUID if you'd like.

Reply:
I was getting an error message that UUIDs may not match in other replicas, so I decided to take a more deterministic approach.